### DISCUSSION TITLE
Micronaut: API functional testing framework multi-module with Junit5

### DISCUSSION THREAD

**Discussion by mukilansai** [micronaut-projects/micronaut-core#10711] [Q&A]
Hello, 
   I have a requirement to create a test framework using Junit5 and micronaut. Eventually all team members would contribute to writing test libraries (a different gradle module). Do I need micronaut application context to load all beans referred by the test module (imported module)? I remember Spring has some mechanism to initialize/use beans in a object. Would micronaut also support?
Thanks, 
Mukil

**Comment by sdelamo** [comment]
I am not sure I follow you. But starting a Micronaut Application context will pick every bean available. You don't have to do anything special.
